Gemfile.local

```diff
@@ -0,0 +1,2 @@
+# replace google with your oauth gem here
+gem 'omniauth-google-oauth2'
```

Comment:
I do agree. I just put it to keep backward compatibility. If we don't keep `Gemfile.local` we have to be sure that we can remove `omniauth-google-oauth2`. Without it won't work out of the box.